static/js/publisher/form.js

```diff
@@ -146,13 +146,6 @@ function initForm(config, initialState, errors) {
 
   let state = JSON.parse(JSON.stringify(initialState));
 
-  const stateInput = document.createElement("input");
```

Comment:
so we no longer need the state as an input, for any values?

Reply:
We don't need state input for saving. Preview uses it's own state input, so this one can be removed.